lib/livebook/hubs/provider.ex

```diff
@@ -30,4 +36,28 @@ defprotocol Livebook.Hubs.Provider do
   """
   @spec connected?(struct()) :: boolean()
   def connected?(struct)
+
+  @doc """
+  Disconnects the given struct.
```

Comment:
?

```suggestion
  Disconnects the given hub.
```

same in others